tests/test_project/admin.py

```diff
@@ -1,12 +1,35 @@
 from django.contrib import admin
-from openwisp_utils.admin import ReadOnlyAdmin
+from django.contrib.admin import ModelAdmin
+from django.contrib.admin.options import InlineModelAdmin
+from django.forms import ModelForm
+from openwisp_utils.admin import AlwaysHasChangedMixin, ReadOnlyAdmin
 
-from .models import RadiusAccounting
+from .models import Id, Operator, RadiusAccounting
+
+
+class OperatorAdmin(ModelAdmin):
+    list_display = ['first_name', 'last_name']
 
 
 class RadiusAccountingAdmin(ReadOnlyAdmin):
     list_display = ['session_id', 'username']
     fields = ['session_id', 'username']
 
 
+class OperatorForm(AlwaysHasChangedMixin, ModelForm):
+    pass
+
+
+class OperatorInline(InlineModelAdmin):
```

Comment:
change to `class OperatorInline(admin.StackedInline):`
this is what is causing the exception